tests/compat/test-compat.sh

```diff
@@ -195,7 +195,7 @@ run_test() {
         )
         suite_path="old_suite/tests/suites"
     fi
-    ./tests/databend-test --suites "$suite_path" --mode 'standalone' --run-dir 0_stateless -- 05_
```

Comment:
Seems `test-compat.sh` was originally intended to run only 05_ddl, but `-- 05_` will match all test cases contains `05_`, so I changed it to `^05_*` to only match test cases in `05_ddl` @drmingdrmer

Reply:
nice